airflow/auth/managers/base_auth_manager.py

```diff
@@ -50,6 +50,10 @@ def get_cli_commands() -> list[CLICommand]:
         """
         return []
 
+    def get_blueprint(self) -> None | Blueprint:
```

Comment:
I would suggest mentioning "api" in the name at least for better understanding

```suggestion
    def get_api_blueprint(self) -> None | Blueprint:
```